word_racer/session.py

```diff
@@ -0,0 +1,295 @@
+import asyncio
+from collections import Counter
+from redbot.core.utils.chat_formatting import box
+from redbot.core.data_manager import cog_data_path
+import discord
+import numpy as np
+import pathlib
+import re
+from PIL import Image, ImageDraw, ImageFont
```

Comment:
Please include a Pipfile change to indicate this dependency. It can be done easily if you have `pipenv` by doing `pipenv install PIL` in the root directory.